package.json

```
@@ -5,11 +5,12 @@
   "scripts": {
     "predev": "check-node-version --package",
     "dev": "next dev",
-    "build": "next build",
+    "build": "next build && next export",
     "start": "next start",
     "lint:js": "next lint",
     "lint:css": "stylelint '**/*.scss'",
-    "lint": "run-s lint:js lint:css",
+    "lint:html": "npm run build && pa11y-ci -c .pa11yci.json './out/**/*.html'",
```

With the other change in place, we can update this as well:

```suggestion
    "lint:html": "npm run export && pa11y-ci -c .pa11yci.json './out/**/*.html'",
```